pages/vi/vi-first-steps.md

```diff
@@ -25,7 +25,7 @@ A very large part of these steps is finding problems with these steps and this M
 There is no official deadline to complete these steps, but most candidates who are approved for the internship program finished the steps within 7-8 days. Good luck!
 
 
-## Step 0 - Introductory Steps
+## Step 0 | Introductory Steps
```

Comment:
This will "kind of" break the link at the bottom of some pages. For example, https://rawgit.com/TooManySpaces/TooManySpaces.github.io/stepFormat/#!./pages/vi/vi-vagrant.md

Maybe we should change the tg-installation page to use `-` since there are less or no links referencing a certain section on that page.

Reply:
Is this the link you're talking about? This link seems to be broken on the master branch as well.

Reply:
Nah, I was referring to the link at the bottom of [this page](https://rawgit.com/TooManySpaces/TooManySpaces.github.io/stepFormat/#!./pages/vi/vi-vagrant.md). Where `#!./pages/vi/vi-first-steps.md#Step_2_-_Vagrant_Tutorial` needs to be updated.  Same with [this page](https://rawgit.com/TooManySpaces/TooManySpaces.github.io/stepFormat/#!pages/vi/vi-vagrant.md) and some more pages. Since this will involve changing a few files, that's why I was suggesting changing tg-installation page to use `-`.

Reply:
Oh, okay, thanks for pointing those out to me. Should I make a new PR for that?

Reply:
@TooManySpaces You could just work on the same `stepFormat ` branch, push your changes and they will be reflected here in this PR.